src/AnnouncementBubble/DeletedAnnouncementBubble.less

```diff
@@ -18,6 +18,25 @@
   .text--medium();
 }
 
+// Family Portal styling
+.MessagingBubble--FamilyPortal {
+  .DeletedAnnouncementBubble--container {
+    background-color: @family_portal_quoted_announcement;
+    color: @family_portal_slate;
+    border: unset;
```

Comment:
Can we get by with a `border: 0` here? Since as we discussed in #frontend, IE11 doesn't support `unset`. No worries if not!